SU2_CFD/include/numerics_simd/flow/convection/roe.hpp

```diff
@@ -168,7 +168,7 @@ class CRoeBase : public Base {
 
     VectorDbl<nVar> flux;
     for (size_t iVar = 0; iVar < nVar; ++iVar) {
-      flux(iVar) = kappa * (flux_i(iVar) + flux_j(iVar));
```

Comment:
Can kappa ever be changed?

Reply:
This was a bug that was fixed in the non-simd Roe scheme but was still here somehow.
kappa should only appear in the dissipation term, here we want the average of the fluxes, and anything different from 0.5 would be wrong.

Reply:
So yes kappa can be changed and a higher value results in lower dissipation (someone wrote it like that :shrug:)